addons/mail/static/src/components/composer_text_input/composer_text_input.xml

```diff
@@ -4,37 +4,13 @@
     <t t-name="mail.ComposerTextInput" owl="1">
         <div class="o_ComposerTextInput">
             <t t-if="composer">
-                <t t-if="composer.hasSuggestedPartners">
-                    <div class="o_ComposerTextInput_mentionDropdown" t-att-class="{ 'o-lowPosition': props.hasMentionSuggestionsBelowPosition }">
-                        <div
-                            t-att-class="{
-                                'dropdown': props.hasMentionSuggestionsBelowPosition,
-                                'dropup': !props.hasMentionSuggestionsBelowPosition,
-                            }"
-                        >
-                            <div class="o_ComposerTextInput_mentionDropdownPropositionList dropdown-menu show">
-                                <t t-foreach="composer.mainSuggestedPartners" t-as="mainSuggestedPartner" t-key="mainSuggestedPartner.localId">
-                                    <PartnerMentionSuggestion
-                                        isActive="mainSuggestedPartner === composer.activeSuggestedPartner"
-                                        partnerLocalId="mainSuggestedPartner.localId"
-                                        t-on-o-partner-mention-suggestion-clicked="_onPartnerMentionSuggestionClicked"
-                                        t-on-o-partner-mention-suggestion-mouse-over="_onPartnerMentionSuggestionMouseOver"
-                                    />
-                                </t>
-                                <t t-if="composer.mainSuggestedPartners.length > 0 and composer.extraSuggestedPartners.length > 0">
-                                    <div role="separator" class="dropdown-divider"/>
-                                </t>
-                                <t t-foreach="composer.extraSuggestedPartners" t-as="extraSuggestedPartner" t-key="extraSuggestedPartner.localId">
-                                    <PartnerMentionSuggestion
-                                        isActive="extraSuggestedPartner === composer.activeSuggestedPartner"
-                                        partnerLocalId="extraSuggestedPartner.localId"
-                                        t-on-o-partner-mention-suggestion-clicked="_onPartnerMentionSuggestionClicked"
-                                        t-on-o-partner-mention-suggestion-mouse-over="_onPartnerMentionSuggestionMouseOver"
-                                    />
-                                </t>
-                            </div>
-                        </div>
-                    </div>
+                <t t-if="composer.hasSuggestions">
+                    <ComposerDropList
+                        composerLocalId="props.composerLocalId"
+                        extraListName="composer.extraListName"
+                        mainListName="composer.mainListName"
+                        activeElementName="composer.activeElementName"
```

Comment:
You can just have `composerLocalId` as prop here, all the others can be read directly on composer record.